fs/vfs/fs_read.c

```diff
@@ -32,37 +32,110 @@
 #include <errno.h>
 
 #include <nuttx/cancelpt.h>
+#include <nuttx/fs/uio.h>
 
 #include "notify/notify.h"
 #include "inode/inode.h"
 
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: iovec_compat_readv
+ *
+ * Description:
+ *   Emulate readv using file_operation::read.
+ *
+ ****************************************************************************/
+
+static ssize_t iovec_compat_readv(FAR struct file *filep,
```

Comment:
```suggestion
static ssize_t file_writev_compat(FAR struct file *filep,
```
let's apply the similar suggestion from fs_write.c

Reply:
done